cmd/query/app/static_handler.go

```diff
@@ -126,6 +126,34 @@ func loadIndexBytes(open func(string) (http.File, error), options StaticAssetsHa
 	return indexBytes, nil
 }
 
+func (sH *StaticAssetsHandler) configListener(watcher *fsnotify.Watcher) {
+	for {
+		select {
+		case event := <-watcher.Events:
+			// Ignore if the event filename is not the UI configuration or if it is a chmod event.
+			if filepath.Base(event.Name) != filepath.Base(sH.options.UIConfigPath) || event.Op&fsnotify.Chmod == fsnotify.Chmod {
+				continue
+			}
```

Comment:
no need to combine unrelated checks
```suggestion
			if filepath.Base(event.Name) != filepath.Base(sH.options.UIConfigPath) {
				continue
			}
			if event.Op&fsnotify.Chmod == fsnotify.Chmod {
				continue
			}
```

Reply:
Ah, I just want to have a more compact code. but sure I'll separe it.